.github/workflows/build.yml

```diff
@@ -2,6 +2,10 @@ name: "Build"
 
 on: [workflow_call]
 
+concurrency:
```

Comment:
The build yml is integrated into `control.yml` therefore I don't think this is necessary here as it should be controlled by control.yml. See the `on workflow_call`.